### ISSUE TITLE
Reorganize the packages page?

### ISSUE THREAD

**Issue by karthik** [ropensci/roweb#191]
It looks like a very long list but there are many deprecated, early development, non existent packages. Also packages like spocc = c(AntWeb, Ecoengine, gbif, inat, rebird) + taxize. If we grouped them, people will know installing spocc + taxize = the biodiversity suite.

**Comment by richfitz** [comment]
:+1: with this and #190.  I wonder if there's some general classification that allows for "published" packages vs experiments / dead ends etc to be distinguished.  It might be worth systematically working through which packages would actually pass through onboarding and only really promote those?

**Comment by sckott** [comment]
Hmm, my take on this is that I want as many eyes as possible on pkgs in early dev - instead of waiting until e.g., 1st version on CRAN, then putting up on pkgs page, - Disagree?

In terms of pkgs that are deprecated/defunct/we-don't-want-people-using, yeah, makes sense to remove those. 

In terms of putting all through onboarding - nice idea, but we can barely get reviewers for the few pkgs we get in onboarding, but I guess we could review each others to speed it up

**Comment by richfitz** [comment]
Definitely. But perhaps a second page "rOpenSci labs" or "rOpenSci beta" could help distinguish between packages that go well and those that are more experimental?

**Comment by sckott** [comment]
Sounds good to separate them, or at least by default show more stable ones, and then toggle a button to see stable + beta on same page

**Comment by cboettig** [comment]
:+1:  for the re-organization and for distinguishing between beta, in production, and deprecated.  

We want people to experiment with the betas but not mistake their potential bugs as representative of more polished stuff.  Also, perhaps we should document the deprecation somehow, rather than having packages just 'disappear'?

**Comment by sckott** [comment]
> Also, perhaps we should document the deprecation somehow, rather than having packages just 'disappear'? 

Where would that go?  on the packages page?

**Comment by cboettig** [comment]
I guess so. or on a separate page linked from the packages page.

On Sun, Jul 19, 2015, 7:54 AM Scott Chamberlain <notifications@github.com>
wrote:

> Also, perhaps we should document the deprecation somehow, rather than
> having packages just 'disappear'?
>
> Where would that go? on the packages page?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ropensci/roweb/issues/191#issuecomment-122667706>.
>
-- 

http://carlboettiger.info

**Comment by cboettig** [comment]
Related to this: https://github.com/ropensci/drat/issues/1

also, really minor issue, but the text describing each topic area (e.g. "Data Publication") is really light-coloured, almost invisible.  Maybe a slightly darker grey (e.g. matching the footer grey) would be better.

**Comment by sckott** [comment]
@cboettig tried, but can't figure out where the color is for those elements - above my css paygrade

**Comment by mbjones** [comment]
@sckott @cboettig Seems to come from flat-ui.css, line 477.  Chrome's "Inspect Element" is super useful for CSS experimentation.

**Comment by sckott** [comment]
@mbjones but I think it comes from a .less file https://github.com/ropensci/roweb/blob/master/assets/flat-ui/less/flat-ui.less

**Comment by karthik** [comment]
> @cboettig tried, but can't figure out where the color is for those elements - above my css paygrade

I can fix those.
